src/main/charts/confluence/values.yaml

```diff
@@ -608,6 +608,31 @@ confluence:
     #
     usePodNameAsClusterNodeName: true
 
+  # Use AWS S3 to store attachments. From Confluence 8.1 onwards
+  #
+  s3AttachmentsStorage:
+    # Bucket name to store attachments. If a bucket name is defined, Confluence will automatically
+    # use AWS S3 to store attachments. Only bucket name is required, not the full arn. This can also be defined in
+    # `confluence.additionalEnvironmentVariables` as `CONFLUENCE_FILESTORE_ATTACHMENTS_S3_BUCKET_NAME`
+    # or `confluence.additionalJvmArgs` as `-Dconfluence.filestore.attachments.s3.bucket.name=<bucket-name>`
+    #
+    bucketName:
+    # AWS region where the S3 bucket is located. This can also be defined in
+    # `confluence.additionalEnvironmentVariables` as `CONFLUENCE_FILESTORE_ATTACHMENTS_S3_BUCKET_REGION`
+    # or `confluence.additionalJvmArgs` as `-Dconfluence.filestore.attachments.s3.bucket.region=<aws-region>`
+    #
+    awsRegion:
+
+    # Use EKS IRSA. See: https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html
+    #
+    eksIrsa:
```

Comment:
Is this useful only for S3? Can we extract that to higher level?

Reply:
perhaps, it should be in serviceAccount section

Reply:
Moved to serviceAccount